playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_INSTRUCTOR_QUERY_PROBLEM_TYPES:
+  - 'problem',
+  - 'openassessment',
+  - 'submit-and-compare',
+  - 'freetextresponse',
+  - 'xblockmufi',
+  - 'qualtricssurvey',
```

Comment:
@sigberto I guess my confusion (here and above) is that I suspect we don't want/need to include all of these problem types.

While "qualtricssurvey" is technically a problem, I wasn't aware of it having a grade or any state,
so I'm not sure what exactly instructors would want to (or be able to) query.

We can touch base with @mondiaz for clarification.